contracts/amm/pool/Joseph.sol

```diff
@@ -216,24 +216,20 @@ abstract contract Joseph is JosephInternal, IJoseph {
     ) internal {
         uint256 autoRebalanceThreshold = _getAutoRebalanceThreshold();
 
-        if (autoRebalanceThreshold > 0) {
-            if (wadOperationAmount >= autoRebalanceThreshold) {
-                _withdrawFromStanleyBeforeRedeem(
-                    milton,
-                    wadMiltonErc20Balance,
-                    vaultBalance,
-                    wadOperationAmount
-                );
-            } else {
-                if (wadOperationAmount > wadMiltonErc20Balance) {
-                    _withdrawFromStanleyBeforeRedeem(
-                        milton,
-                        wadMiltonErc20Balance,
-                        vaultBalance,
-                        wadOperationAmount
-                    );
-                }
-            }
+        if (wadOperationAmount > wadMiltonErc20Balance) {
```

Comment:
Maybe we can do in one if statement
```
if( (wadOperationAmount > wadMiltonErc20Balance) || (autoRebalanceThreshold > 0 && wadOperationAmount >= autoRebalanceThreshold) ) 
```
?

Reply:
good point, fixed